terraform/output.tf

```diff
@@ -0,0 +1,4 @@
+output "outbound_ip_ranges" {
+  value       = sort([for ip in azurerm_linux_web_app.main.possible_outbound_ip_address_list : "${ip}/32"])
+  description = "The IP address ranges to use to filter requests to this App Service instance, like for the Eligibility Server"
```

Comment:
I think this comment is... maybe a little confusing? We use these IPs to filter requests _to the Eligibility Server_, from this App Service instance.

Reply:
Whoops, good catch!